### ISSUE TITLE
hiryan

### ISSUE THREAD

**Issue by RoloRotMG** [RotMC/Ephemera#4]
Couldn't get myself to put this under "bug report" because at this point we are working around bugs that can't be fixed due to them causing more severe bugs.

So from now on. Anytime we edit a slime world we will edit it in a 1.17.1 test server. Currently, Terra is our test server for this. It has the essential plugins and is on 1.17.1 ready to go. I have a custom FAWE jar that allows me to take schematics of 1.19.2 builds and paste them onto 1.17.1 with no errors or issues.

So here is what I need from you.
For dungeons, the end environment works great, end environment is also completely fine as long as the build is not in 0,0,0. (An unloaded enderdragon floating around a slime world is not an issue in the grand scheme of things)
However the normal environment causes all blocks to be placed exactly 64 blocks under. I'd manually change the coordinates of all y levels on all dungeon configurations to be -64 but the issue is ONLY blocks are affected. Not tile entities. And whilst this isn't an issue in dungeons. It is an issue in the realm. So the realm is now a 1.17.1 slime world but it is pasted lower down. This messes up worldguard, mythicmobs spawns and worst of all. We get this horrendous spam on startup
https://pastebin.com/MKa6yQF1 <- This pastes once per tile entity, and we have a lot of them.

I tried testing to see what would happen if I set Cartographer to load the realm as the_end or nether environments but the same thing happened which also makes me believe that those environments don't work with Cartographer, it just defaults to normal. But I assume if I set it to nether or the_end it would be fine (you gotta fix this for shadow realm)

TL;DR do something to make anything in overworld manually place the blocks 64 blocks higher somehow in ephemera and fix cartorapher environments

**Comment by RoloRotMG** [comment]
As discussed, environment was changed to dimension so I set dimension to "minecraft:the_nether" and there are no errors and everything is working fine so assume the end will be fine too. good stuff

**Comment by RoloRotMG** [comment]
All good
